schematools/cli.py

```diff
@@ -187,6 +187,18 @@ def permissions_revoke(db_url, role):
     default=False,
     help="Execute SQL statements or dry-run [default]",
 )
+@click.option(
+    "--read/--no-read",
+    "set_read_permissions",
+    default=True,
+    help="Set read permissions [default=read]",
```

Comment:
Geeft argument `show_default=True` mee en je hoef het niet zelf aan te geven in de help tekst.